libraries/wallet/wallet.cpp

```diff
@@ -1174,7 +1174,71 @@ class wallet_api_impl
       return create_account_with_private_key(owner_privkey, account_name, registrar_account, referrer_account, broadcast, save_wallet);
    } FC_CAPTURE_AND_RETHROW( (account_name)(registrar_account)(referrer_account) ) }
 
-
+   signed_transaction create_custom_authority(account_id_type account,
+                                              int operation_type,
```

Comment:
What is type in comparison to `operation_id`?

Reply:
He probably means operation_id, best to change the variable name

Reply:
Sorry for confusion, I meant operation_type_id. Integer identifier of operation type allowed by custom authority.

Reply:
Usually in bitshares-core we use `xxx_id` to indicate some data stored in the database, e.g. account_id (1.2.x), asset_id (1.3.x) and etc. Due historical reason, `operation_id` is used in `operation_history_object.hpp` to indicate data which is `operation_history_id_type`. Mostly, when we talk about operation IDs, we're actually talking about operation history IDs. Then, when we want to specify one of the operations, we use another word e.g. `operation_type`. Actually, in `api.hpp` and `api.cpp`, both `operation_id` and `operation_type` are used as variable names to specify operations, and the code with `operation_id` as variable name is a bit messy; in `wallet.hpp` and `wallet.cpp` only `operation_type` is used. To be consistent, IMHO, we should refactor the code to use `operation_type` everywhere.

Reply:
Thanks a lot for the clarification @abitmore 

@RomaMokych this is what I meant in the other comment

Reply:
done